src/redux/validation/validation.worker.ts

```diff
@@ -0,0 +1,50 @@
+/// <reference lib="webworker" />
+import log from 'loglevel';
+
+import {ResourceParser, SchemaLoader, createDefaultMonokleValidator} from '@monokle/validation';
+
+import {
+  LoadValidationMessage,
+  LoadValidationMessageType,
+  RunValidationMessage,
+  RunValidationMessageType,
+  WorkerMessage,
+  matchWorkerEvent,
+} from './validation.worker.types';
+
+export const RESOURCE_PARSER = new ResourceParser();
+export const SCHEMA_LOADER = new SchemaLoader();
+export const VALIDATOR = createDefaultMonokleValidator(RESOURCE_PARSER, SCHEMA_LOADER);
```

Comment:
Can you try `createExtensibleMonokleValidator` and hardcode a custom plugin to your configuration? Just to double check if you can dynamically load ESM in Electron's worker.

For example:
```
plugins:
  nsa: true
```

Reply:
I tried it now and I don't see any errors, seems to work fine!